ethcore/src/executive.rs

```diff
@@ -276,7 +276,7 @@ impl<'a, B: 'a + StateBackend> Executive<'a, B> {
 
 			let cost = builtin.cost(data);
 			if cost <= params.gas {
-				builtin.execute(data, &mut output);
+				builtin.execute(data, &mut output)?;
```

Comment:
not sure i follow
why shouldn't it fail the same way as not enough gas fails, but with different reason?